templates/tags/SageFinancialProgramCodesOther.json

```diff
@@ -1,44 +1,48 @@
 [
   "3D/PEC DCC-MSSM / 105000",
-  "Agora / 112300",
   "AMP 2.0 Diverse Cohorts AD Portal / 102300",
   "AMP AD Supp Diversity Cohorts-Columbia / 121300",
   "AMP AD Supp Diversity Cohorts-Mayo / 121400",
   "AMP Longevity Supplement / 102000",
   "ANTI PD1 CH-BMS / 507200",
+  "Agora / 112300",
+  "Allen Institute DREAM CH / 507400",
   "AoU Governance-ELSI / 122901",
   "At Home-Mass Gen Hosp / 120000",
-  "BMGF-Ki / 30144",
```

Comment:
We need to keep some of these around even though they no longer exist in the finance app.  It's because these have been used to tag resources and we need to keep the resources around even after the code disappears.  here's an example: https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/config/prod/gates-ki-001.yaml

Reply:
Khai is right.  It might be safest to make additions only in this PR, omitting deletions. (If you maintain the original sort order you can see the additions and deletions a little easier in the PR.)

Reply:
I've re-added the inactive code, but vim's case-insensitive sorting has moved it down a couple rows